razor-chase/src/chase/impl/basic.rs

```diff
@@ -390,16 +425,26 @@ impl ModelTrait for Model {
 impl fmt::Display for Model {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
         let domain: Vec<String> = self.domain().into_iter().map(|e| e.to_string()).collect();
-        let elements: Vec<String> = self.domain().iter().sorted().iter().map(|e| {
-            let witnesses: Vec<String> = self.witness(e).iter().map(|w| w.to_string()).collect();
-            let witnesses = witnesses.into_iter().sorted();
-            format!("{} -> {}", witnesses.into_iter().sorted().join(", "), e)
-        }).collect();
+        let elements: Vec<String> = self
```

Comment:
huh, I see the opposite change here. I guess the idea is to either break everything into separate lines or don’t break them at all?

Reply:
If combining everything into a single line does not exceed a certain character limit (I believe it's 80 chars), then rustfmt will combine them. Otherwise, it breaks them apart into multiple lines as you can see here.